src/lib/handCheck.js

```diff
@@ -3,13 +3,63 @@ import _ from "lodash";
 //Detects a card's number
 const cardNumber = (card) => card.slice(1, 3);
 
+const advancedCardNumber = (card) => {
+  switch (card) {
+    case "02":
+      return "2";
+    case "03":
+      return "3";
+    case "04":
+      return "4";
+    case "05":
+      return "5";
+    case "06":
+      return "6";
+    case "07":
+      return "7";
+    case "08":
+      return "8";
+    case "09":
+      return "9";
+    case "10":
+      return "10";
+    case "11":
+      return "j";
+    case "12":
+      return "q";
+    case "13":
+      return "k";
+    case "14":
+      return "a";
+    default:
+      return null;
+  }
+};
```

Comment:
This seems like a duplicate from a previous functions. You can make it a signle function and use it in to places

Reply:
Done